app/qml/main.qml

```diff
@@ -402,7 +402,23 @@ ApplicationWindow {
     Connections {
         target: __loader
         onLoadingStarted: projectLoadingScreen.visible = true
-        onLoadingFinished: projectLoadingScreen.visible = false
+        onLoadingFinished:
+        {
+            projectLoadingScreen.visible = false;
+            if ( invalidLayers.length > 0 )
+            {
+                let warningMessage = qsTr( "WARNING: the following layers are invalid: " );
```

Comment:
How about some less prominent message like "We looked everywhere but could not find some of your layers: .... <layers> .... see [documentation to learn how to package them properly](https://help.cloudmergin.com/plugin-new-project?_ga=2.122664654.477250255.1637567361-1897708595.1630658758)" with link to documentation.
@saberraz @wonder-sk do you have any suggestions for the message?